main.go

```diff
@@ -57,12 +49,12 @@ func main() {
 
 	// /api routes allow the React frontent to communicate with the backend
 	r.HandleFunc("/api/comment/{id}", routes.GetComment).Methods("GET")
-	r.HandleFunc("/api/user/{id}", routes.GetUser).Methods("GET")
+	r.HandleFunc("/api/user", routes.GetUsers).Methods("GET")
 	r.HandleFunc("/api/comment", routes.AddComment).Methods("POST")
 
 	// Create an http server using the mux router
 	srv := &http.Server{
-		Handler:      r,
+		Handler:      handlers.CORS()(r),
```

Comment:
Why is CORS necessary here? The API and the React frontend have the same origin, no?

Reply:
Now they don't